scripts/abby_excel.py

```
@@ -0,0 +1,183 @@
+# to average
+ingestion_time = collections.defaultdict(list)
+ingestion_memory = collections.defaultdict(list)
+compression_ratio = collections.defaultdict(list)
+ingestion_speed = collections.defaultdict(list)
+
+for output_dir in outputs:
+    tool = os.path.basename(output_dir.resolve())
+    filename = (output_dir / "output.json").resolve()
+    if not os.path.exists(str(filename)):
+        continue
+    with open(filename, 'r') as file:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)